manifests/w/WaterbugStudios/Mazerator/0.0.2/WaterbugStudios.Mazerator.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WaterbugStudios.Mazerator
+PackageVersion: 0.0.2
+PackageLocale: en-US
+Publisher: Waterbug Studios
```

```suggestion
Publisher: Waterbug Studios/Awesome Arcade
```
cc @AwesomeKalin: This must match Control Panel.